packages/extensions/upward-security-headers/package.json

```diff
@@ -7,20 +7,22 @@
   "description": "Add security headers to UPWARD",
   "main": "intercept.js",
   "scripts": {
-    "clean": " "
+    "clean": "rimraf ./node_modules"
```

Comment:
`clean` seems to be more about removing `dist` in all other packages. Can we keep this line and just remove the `rimraf` dependency entirely?

Reply:
I'm fine with either approach, as long as the bug is fixed. Interested in QA outcome.

Reply:
Hmm - it removes node modules in the gql-queries package


`"clean": "rimraf ./lib/magento-compatibility.js ./node_modules",`